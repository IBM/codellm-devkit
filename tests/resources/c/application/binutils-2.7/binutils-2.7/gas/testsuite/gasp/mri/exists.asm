exists	macro	arg1,arg2
	ifne	==arg2
	move	arg1,arg2
	elsec
	push	arg1
	endc
	endm

	exists	foo,bar
	exists	foo
