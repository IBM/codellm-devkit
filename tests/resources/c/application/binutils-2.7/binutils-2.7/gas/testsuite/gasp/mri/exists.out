;exists	macro	arg1,arg2
;	ifne	==arg2
;	move	arg1,arg2
;	elsec
;	push	arg1
;	endc
;	endm
;

;	exists	foo,bar
;	ifne	-1
;	move	foo,bar
	move	foo,bar
;	elsec
;	push	foo
;	endc
;	exists	foo
;	ifne	0
;	move	foo,
;	elsec
;	push	foo
	push	foo
;	endc
;